ton_client/src/debot/sdk_interface.rs

```diff
@@ -246,38 +294,10 @@ const ABI: &str = r#"
 			]
 		},
 		{
-			"name": "encrypt",
+			"name": "constructor",
```

Comment:
looks strange

Reply:
I've recompiled abi file in consortium and copied it here. Maybe some functions have changed positions in file.